core/src/test/scala/unit/kafka/utils/TestUtils.scala

```diff
@@ -1433,6 +1448,8 @@ object TestUtils extends Logging {
         val cause = e.getCause()
         assertTrue("Expected an exception of type " + clazz.getName + "; got type " +
             cause.getClass().getName, clazz.isInstance(cause))
+        errorMessage.foreach(message => assertTrue(s"Wrong message ${cause.getMessage}", cause.getMessage.contains(message)))
```

Comment:
nit: Could we change the error message to be more informative? e.g `Message X did not contain message Y as expected"
I also suggest we change the `errorMessage` variable name to `errorMessageSubset` (or similar) to make it more obvious that we want the message to be contained and not directly equal the message we get. 

The method name `assertFutureExceptionTypeEquals` would leave me to expect that we want everything to be 1:1 equal otherwise

Reply:
for now, i updated the message.